react/ProductList.tsx

```diff
@@ -41,6 +45,21 @@ const ProductList: FC = () => {
     removeItem({ uniqueId })
   }
 
+  if (renderAsChildren) {
+    return (
+      <div
+        className={`w-100 h-100 overflow-y-auto ${handles.minicartProductListContainer}`}
```

Comment:
Isn't it easier to change the classes if the prop is true instead of creating this whole if?

Reply:
Yep!